### PR TITLE
nss and nspr package updates

### PR DESCRIPTION
**Pull request by heitbaum**
Update nss from 3.47.1 to 3.60 

- no longer building libcrmf.a
- see release notes. A fair bit of change.
- https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS/NSS_Releases
- The legacy DBM database, libnssdbm, is no longer built by default.
- scripts/build inputstream.adaptive Has built against this nss version.

leaving this as draft. It needs some further testing. It is currently running on Generic

### CODE REVIEW

**Comment by heitbaum** [comment]
Been running this for the week now

File: packages/security/nss/package.mk [hunk 1]
```diff
@@ -35,7 +36,7 @@ makeinstall_host() {
   cd $PKG_BUILD
   $STRIP dist/Linux*/lib/*.so
   cp -L dist/Linux*/lib/*.so $TOOLCHAIN/lib
-  cp -L dist/Linux*/lib/libcrmf.a $TOOLCHAIN/lib
+  #cp -L dist/Linux*/lib/libcrmf.a $TOOLCHAIN/lib
```
Comment (lrusak) [inline packages/security/nss/package.mk:39]: remove rather than comment

File: packages/security/nss/package.mk [hunk 2]
```diff
@@ -70,7 +72,7 @@ makeinstall_target() {
   cd $PKG_BUILD
   $STRIP dist/Linux*/lib/*.so
   cp -L dist/Linux*/lib/*.so $SYSROOT_PREFIX/usr/lib
-  cp -L dist/Linux*/lib/libcrmf.a $SYSROOT_PREFIX/usr/lib
+  #cp -L dist/Linux*/lib/libcrmf.a $SYSROOT_PREFIX/usr/lib
```
Comment (lrusak) [inline packages/security/nss/package.mk:75]: remove rather than comment

**Review by lrusak — COMMENTED** [review]
Probably fine after the minors

File: packages/security/nss/package.mk (hunk 1 above)
Comment (heitbaum) [reply packages/security/nss/package.mk:39]: Fixed

File: packages/security/nss/package.mk (hunk 2 above)
Comment (heitbaum) [reply packages/security/nss/package.mk]: Fixed.

File: packages/security/nss/package.mk (hunk 1 above)
Comment (lrusak) [reply packages/security/nss/package.mk:39]: It's still here :)

File: packages/security/nss/package.mk [hunk 3]
```diff
@@ -3,11 +3,11 @@
...
 
 PKG_NAME="nss"
-PKG_VERSION="3.47.1"
-PKG_SHA256="07d4276168f59bb3038c7826dabb5fbfbab8336ddf65e4e6e43bce89ada78c64"
+PKG_VERSION="3.60"
+PKG_SHA256="850deb3f4d49703618b64ca5c0ca92d2f195e0e41d7124f7b2149a6ca9eaae1f"
 PKG_LICENSE="Mozilla Public License"
 PKG_SITE="http://ftp.mozilla.org/"
-PKG_URL="http://ftp.mozilla.org/pub/security/nss/releases/NSS_3_47_1_RTM/src/nss-3.47.1-with-nspr-4.23.tar.gz"
+PKG_URL="https://ftp.mozilla.org/pub/security/nss/releases/NSS_3_60_RTM/src/nss-3.60-with-nspr-4.29.tar.gz"
```
Comment (CvH) [inline packages/security/nss/package.mk:10]:
```suggestion
PKG_URL="https://ftp.mozilla.org/pub/security/nss/releases/NSS_${PKG_VERSION/./_}_RTM/src/nss-${PKG_VERSION}-with-nspr-$(get_pkg_version nspr).tar.gz"
```
:)

**Review by lrusak — APPROVED** [review]
LGTM

**Pull request merged**